### PR TITLE
Dat 2909 - New views for portable infoboxes

### PR DESCRIPTION
**Pull request by RafLeszczynski**
Please do not merge, this is a part of bigger story.

### CODE REVIEW

**Comment by dianafa** [comment]
LGTM :+1:

File: front/styles/main/module/article/_portable-infobox.scss [hunk 1]
```diff
@@ -13,60 +27,126 @@ $infobox-section-header-background: #e5e5e5;
...
-		font-size: 1rem;
-		font-weight: bold;
-		margin-top: 0;
+	//////////// Specific styles for each of the infobox elements starts here ////////////
+
+	// infobox hero
+	.item-type-hero {
+		position: relative;
+
+		// TODO: should be removed ones the default caption is removed from image in hero
```
Comment (rogatty) [inline front/styles/main/module/article/_portable-infobox.scss]: Do we have a ticket for doing that? If not, can you create one and put its number here?

Comment (dianafa) [inline front/styles/main/module/article/_portable-infobox.scss]: it's already done in ticket:

**Comment by rogatty** [comment]
Please merge the latest dev and see if unit tests pass.

**Comment by SebastianMarzjan** [comment]
:+1:

**Pull request closed without merging**